monitoring/shutdown-javaagent/pom.xml

```diff
@@ -0,0 +1,191 @@
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>2.4</version>
+        <executions>
+          <execution>
+            <phase>process-test-classes</phase>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <finalName>shutdown-javaagent-test</finalName>
+              <archive>
+                <manifest>
+                  <addClasspath>true</addClasspath>
+                  <classpathPrefix>${project.build.directory}/lib</classpathPrefix>
+                </manifest>
+                <manifestEntries>
+                  <Premain-Class>com.google.cloud.runtimes.monitoring.shutdown.Agent</Premain-Class>
+                </manifestEntries>
+              </archive>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+      <artifactId>maven-resources-plugin</artifactId>
+      <executions>
+      <execution>
+        <id>copy-agent-jars</id>
+        <phase>generate-resources</phase>
+        <goals>
+          <goal>copy-resources</goal>
+        </goals>
+        <configuration>
+          <outputDirectory>${project.agentjars.dir}</outputDirectory>
+          <resources>
+            <resource>
+              <directory>src/main/resources/scripts</directory>
+              <includes>
+                <include>**</include>
+              </includes>
+            </resource>
+          </resources>
+        </configuration>
+      </execution>
+      </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy</id>
+            <phase>process-test-classes</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>
+                ${project.build.directory}/lib/
+              </outputDirectory>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-artifact</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>${project.groupId}</groupId>
+                  <artifactId>${project.artifactId}</artifactId>
+                  <version>${project.version}</version>
+                  <type>${project.packaging}</type>
+                  <outputDirectory>${project.agentjars.dir}</outputDirectory>
+                  <destFileName>shutdown-javaagent.jar</destFileName>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```

Comment:
You might wish to add some of the code review tools from java-repo-tools.  (CheckStyle and more, and ErrorProne)